heartbeat/_meta/fields.common.yml

```diff
@@ -172,6 +172,9 @@
               type: text
             - name: id
               type: keyword
+            - name: tags
+              description: Tags used for grouping journeys
+              type: object
```

Comment:
I don't think this is right. Since it's a list of strings it should be `keyword`. As a reminder ES mappings have no list type since any field can be multi-valued.